ocaml/rrdreader.ml

```diff
@@ -25,6 +25,14 @@
 
```

Comment:
I think you should move this utility into the rrd-transport package and install the binary from there.   You need to depend on rrd-transport anyway, and if you do this you can remove the OCaml dependency from this library.   The reader utility may also be useful to other users of rrd-transport.